### ISSUE TITLE
Compatiple with Biome's O Plenty?

### ISSUE THREAD

**Issue by smallerYeet** [rehwinkel/treemendous#4]
I really want to play this mod, but it just seems that there are no Biomes spawning when Biome's O Plenty generation is turned on.
If its possible I would like to have both!

Mc Version: 1.16.3
Forge Version: 34.1.42
Mod Version: 0.4.3

**Comment by Griefed** [comment]
My word generates Treemendous biomes and trees with the Biomes O'Plenty worldtype enabled. Maybe it got fixed since you've opened this ticket?
![2021-01-06_07 26 56](https://user-images.githubusercontent.com/44273438/103736536-c02f4200-4ff0-11eb-8866-207b608c1947.png)

MC Version: 1.16.4
Forge Version: 35.1.28
Mod Version: 0.4.3

**Comment by willrun4fun** [comment]
BYG work to?

**Comment by Griefed** [comment]
Yep, BYG and BOP work together in the overworld out of the box, so to speak.
In order for BYG to generate it's biomes in the Nether and End whilst using the biomesoplenty world type however, you have two choices:
1. Edit the `level.dat` of an already existing world with NBTExplorer
2. Use the `Import Settings` feature of Minecraft during world creation

Here's a guide which explains both ways: https://wiki.griefed.de/SCP/edit-world
